keyboards/crkbd/rev2/info.json

```
@@ -0,0 +1,233 @@
+{
+    "usb": {
+        "pid": "0x0002"
+    },
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "lto": true,
+        "mousekey": true,
+        "nkro": true,
+        "encoder": true,
+        "oled": true,
+        "rgblight": false,
+        "rgb_matrix": true
```

These are all already enabled at the keyboard level `info.json`.
```suggestion
        "encoder": true,
        "rgb_matrix": true
```